imkar/utils.py

```diff
@@ -0,0 +1,65 @@
+"""Utilities for imkar."""
+import numpy as np
+import pyfar as pf
+
+
+def paris_formula(coefficients, incident_directions):
+    r"""
+    Calculate the random-incidence coefficient
+    according to Paris formula.
+
+    The implementation follows the Equation 2.53 from [#]_ and is
+    discretized as:
+
+    .. math::
+        c_{rand} = \sum_{\Omega_S} c(\Omega_S) \cdot |\Omega_S \cdot n| \cdot w
+
+    with the ``coefficients`` :math:`c`, and the
+    area weights :math:`w` from the ``incident_directions``.
+    :math:`|\Omega_S \cdot n|` represent the cosine of the angle between the
+    surface normal and the incident direction.
```

Comment:
This is very implicit. Can you please be more verbose here and also add a check for `np.all(incident_directions.z >= 0`?